ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/redhat-chaos-pod-scenarios-etcd-disruption-commands.sh

```diff
@@ -22,6 +22,7 @@ export KUBECONFIG=/tmp/config
 
 export KRKN_KUBE_CONFIG=$KUBECONFIG
 export NAMESPACE=$TARGET_NAMESPACE 
+export TELEMETRY_PASSWORD=$(cat "/secrets/telemetry/telemetry_password")
```

Comment:
here i think you need secret singular. based on the mount_path below

Reply:
Fixed it. Thanks.